homeassistant_satellite/ha_connection.py

```diff
@@ -38,8 +38,11 @@ def __init__(self, host: str, protocol: str, token: str, port: int = 8123):
 
         ws_protocol = "wss" if protocol == "https" else "ws"
         url = f"{ws_protocol}://{host}:{port}/api/websocket"
+        if skipVerify:
+            self._session = aiohttp.ClientSession(connector=aiohttp.TCPConnector(verify_ssl=False))
+        else:
+            self._session = aiohttp.ClientSession()
 
-        self._session = aiohttp.ClientSession()
```

Comment:
Why don't you always pass the negated value to the `TCPConnector` here?

Reply:
done